Compile error on xenial

```
/<<BUILDDIR>>/mir-0.31.2+dev331-ga18a602f4a/tests/unit-tests/test_logind_console_services.cpp:210:59: error: ignoring return value of ‘ssize_t write(int, const void*, size_t)’, declared with attribute warn_unused_result [-Werror=unused-result]
                 ::write(STDOUT_FILENO, buffer, bytes_read);
                                                           ^
```
https://launchpadlibrarian.net/371347404/buildlog_ubuntu-xenial-ppc64el.mir_0.31.2+dev331-ga18a602f4a-0ubuntu16.04_BUILDING.txt.gz

Affects (at least)  ppc64el, i386 & amd64

--- a/tests/unit-tests/test_logind_console_services.cpp
+++ b/tests/unit-tests/test_logind_console_services.cpp
@@ -207,7 +207,7 @@ public:
             std::cout << "Messages from DBusMock: " << std::endl;
             while ((bytes_read = ::read(mock_stdout, buffer, sizeof(buffer))) > 0)
             {
-                ::write(STDOUT_FILENO, buffer, bytes_read);
+                (void)::write(STDOUT_FILENO, buffer, bytes_read);
             }
             if (bytes_read < 0)
             {

